slides/lecture8.md

```
@@ -13,6 +13,21 @@ Webエンジニアになろう講習会 第8回
 
 ---
 
+# 自己紹介
+
+<div class="columns">
+  <div>
+    <img src="assets/lecture8/icon.png"/>
```

markdownlint-cli2 (0.23.2)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)